app/services/forecast/import.rb

```diff
@@ -1,5 +1,4 @@
 require "csv"
-require "set"
```

Comment:
Are these no longer required because of the Ruby upgrade?

Reply:
Yes that's right, this is deleted when setup runs with the new version